examples/openai/azure_openai.py

```diff
@@ -0,0 +1,50 @@
+import mlflow
+import openai
+import os
+import pandas as pd
+
+os.environ["OPENAI_API_KEY"] = "<YOUR AZURE OPENAI KEY>"
+os.environ["OPENAI_API_BASE"] = "<YOUR AZURE OPENAI BASE>"
+os.environ["OPENAI_API_VERSION"] = "2023-05-15"
+os.environ["OPENAI_API_TYPE"] = "azure"
+
+with mlflow.start_run():
+    model_info = mlflow.openai.log_model(
+        # For Azure OpenAI, model doesn't matter in practice
```

Comment:
Isn't this mapped to `engine` and will select among the different offerings?

Reply:
I'm not sure about this, I can remove the comment. But looks like in order to use Azure OpenAI you need to deploy a model first, and that deployment_id/name is used against request.